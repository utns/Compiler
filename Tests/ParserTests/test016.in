var
    i, j: integer;
    a: array[1..5] of array[1..10] of integer;
begin
    for i := 1 to 5 do
        for j := 1 to 10 do
            a[i, j] := i * j;    
end.
