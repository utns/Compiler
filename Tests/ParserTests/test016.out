begin
for i
____1
to
____5
for j
     ____1
___to
     ___10
               ____j
          ____*
               ____i
     ___:=
               ____j
               ____i
          ____[
               ____a
end
